ntp-udp/src/lib.rs

```diff
@@ -67,6 +85,23 @@ impl UdpSocket {
         }
     }
 
+    async fn fetch_send_timestamp(&self) -> io::Result<Option<NtpTimestamp>> {
+        trace!("waiting for socket to become readable");
+        loop {
+            let mut guard = self.io.readable().await?;
+            match guard.try_io(|inner| fetch_send_timestamp_help(inner.get_ref())) {
+                Ok(send_timestamp) => {
+                    dbg!(&send_timestamp);
```

Comment:
This should be replaced with proper logging

Reply:
should it? it was there for development, to see that we actually ended up in that branch. Do you think it's valuable in production?